src/components/PeopleInfo/PeopleInfo.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+
+import cn from 'classnames';
+import { Person } from '../../types/Person';
+
+type Props = {
+  person: Person;
+  onSelect?: (
+    event: React.MouseEvent<HTMLAnchorElement, MouseEvent>,
+    person: Person
+  ) => void;
+};
+
+export const PeopleInfo: React.FC<Props> = ({
+  person,
+  onSelect = () => {},
+}) => {
+  const { sex, name } = person;
+
+  return (
+    <div className="dropdown-item">
+      <a
+        href="/"
+        onClick={(event) => {
+          onSelect(event, person);
+        }}
```

Comment:
why do we need to pass event here to onSelect, we can just call event.preventDefault() in this component
```suggestion
        onClick={(event) => {
          event.preventDefault();
          onSelect(person);
        }}
```